### PR TITLE
Implemented task and tests

### PR DESCRIPTION
**Pull request by ekreed**
- [DEMO LINK](https://ekreed.github.io/layout_stars/)
- [TEST REPORT LINK](https://ekreed.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,12 +2,65 @@
...
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star stars__star--active"></div>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:26]:
```suggestion
      <div class="stars__star"></div>
```
no need modifier for stars__star 
for amount of active stars should be responsible stars--n modifier

File: src/index.html [hunk 2]
```diff
@@ -2,12 +2,65 @@
...
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars__star stars__star--active"></div>
+      <div class="stars__star stars__star--active"></div>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:35]:
```suggestion
      <div class="stars__star"></div>
      <div class="stars__star"></div>
```
same here and for other stars rows

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star--active {
```
Comment (VitaliyBondarenko1982) [inline src/style.css:18]: instead modifier for  .stars__star--active modifier use .stars-1 ... .stars-5 modifiers and `:nth-child` pseudo class

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by volodymyr-soltys97 — APPROVED** [review]
Nice work 🔥

**Pull request closed without merging**